src/neph/program.cr

```diff
@@ -9,6 +9,13 @@ class Program
     opt_parser.banner = "Usage: neph [options] [job name]"
     opt_parser.on "-v", "--version", "Show software version" { STDERR.puts VERSION; exit 0 }
     opt_parser.on "-h", "--help", "Show this help message" { STDERR.puts opt_parser; exit 0 }
+    opt_parser.on "-f", "--file", "Specify an alternative file" do |config|
+      unless File.exists?(config)
```

Comment:
In this case it should `raise OptionParser::Exception.new`, instead of exiting directly, because the block that catches the exception determines what exit code should be used (different exit codes for different errors), and also formats the message. See how other errors are handled in argument parsing.